neevaco/neeva-ios
# Expand LSApplicationQueriesSchemes + Fix OpenInAppOverlay

We already have the code to support bouncing URLs, just needed to expand the list of URLs to bounce. Happy to take suggestions of more to add, we get a maximum of 50. Also, fixed a bug where the `OpenInAppOverlay` would not dismiss.

## Checklists
### How was this tested?

### Manual test cases
- Go to google maps
- Open Lyft link
 - Make sure `OpenInAppOverlay` appears

## Issues addressed
Fixed: #3198
